core/storage/user/gae_models.py

```diff
@@ -2497,9 +2708,22 @@ def get_export_policy(cls):
         """
         return dict(super(cls, cls).get_export_policy(), **{
             'gae_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'parent_user_id': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'parent_user_id': base_models.EXPORT_POLICY.EXPORTED
         })
 
+    @staticmethod
+    def export_data(user_id):
+        """Exports the username of the parent."""
+        user_auth_model = UserAuthDetailsModel.get_by_id(user_id)
+        if user_auth_model and user_auth_model.parent_user_id:
+            parent_data = UserSettingsModel.get(user_auth_model.parent_user_id)
+            parent_username = parent_data.username
+            return {
+                'parent_user_id': parent_username
```

Comment:
This seems wrong? (as previously discussed). The key shouldn't be "user id"

Reply:
done